master/reference/node/configuration.md

```diff
@@ -31,6 +31,7 @@ The `calico/node` container is primarily configured through environment variable
 | CALICO_IPV4POOL_NAT_OUTGOING | Controls NAT Outgoing for the IPv4 Pool created at start up. | boolean | true |
 | CALICO_IPV6POOL_NAT_OUTGOING | Controls NAT Outgoing for the IPv6 Pool created at start up. | boolean | false |
 | CALICO_STARTUP_LOGLEVEL      | The log severity above which startup calico/node logs are sent to the stdout. | string | ERROR |
+| CLUSTER_TYPE | Contains comma delimited list of indicators about this cluster.  Expected to contain information like hosted, services, kubeadm, tectonic, or datastore type. If running KDD then KDD will automatically be added to this list. | string | |
```

Comment:
The first KDD perhaps should be "Kubernetes API as the Datastore" since we don't use the term KDD in our docs.